src/Main.css

```diff
@@ -81,79 +81,63 @@ body {
 }
 
 /* Armory Page Styling */
-.armoryright {
-    margin: 100px 0px 0px 20px;
+.armoryleft {
+    width: 30%;
+    padding-left: 20px;
 }
 
-.armoryright h6 {
+.armoryleft h6 {
     margin-top: 50px;
 }
 
-.armorymiddle {
-
+.armoryleft h3 {
+    margin-top: 150px;
 }
 
-.armoryleft {
-
+.armorymiddle {
+    width: 40%;
+    text-align: center;
 }
 
-/* Radio Button and Checkbox Styling */
-.defaultRadioButton {
-    display: block;
-    position: relative;
-    padding-left: 10px;
-    margin-bottom: 12px;
-    cursor: pointer;
-    font-size: 22px;
-    user-select: none;
-    -webkit-user-select: none;
-    -moz-user-select: none;
-    -ms-user-select: none;
+.armorymiddle h6 {
+    padding-top: 450px;
 }
 
-.defaultRadioButton input {
-    position: absolute;
-    opacity: 0;
-    cursor: pointer;
-    height: 0;
-    width: 0;
+.armorymiddle h1 {
+    margin-top: 150px;
 }
 
-.checkmark {
-    position: absolute;
-    top: 0;
-    left: 0;
-    height: 25px;
-    width: 25px;
-    background-color: #04CCFF;
-    border-radius: 50%;
+.armoryright {
+    width: 30%;
+    margin-top: 150px;
+    padding-right: 20px;
 }
 
-.defaultRadioButton:hover input ~ .checkmark {
-    background-color: rgb(1, 150, 187);
+/* Training Arena Styling */
+.taleft {
+    width: 30%;
+    margin-top: 150px;
+    padding-left: 20px;
```

Comment:
Instead of all the padding and width play we could use bootstrap. It takes things like that for us. If we want to talk about it tomorrow I'd be down to have a conversation about this.